include/tcpdf_static.php

```diff
@@ -1981,7 +1981,7 @@ public static function fileGetContents($file) {
 			    return $ret;
 			}
 			// try to use CURL for URLs
-			if (!ini_get('allow_url_fopen')
+			if (ini_get('allow_url_fopen')
```

Comment:
For me this has nothing to do with checking curl is available or not

Reply:
I guess you're right. The issue should probably be titled 'Fixes if url's can be accessed by curl'. Be me guest and change it. The fix is valid nevertheless though.

Reply:
I can not understand why `allow_url_fopen` is checked since it has nothing to do with curl ?
Or is it to make a priority on `url_fopen` over curl ?

Reply:
To be honest, I don't know. The only thing I know is that it works as expected, after making this change change :)

Reply:
It's not my project and the code is far to complex (understatement) to analyze how this issue could be fixed in a way you see fit. Besides, it does not seem worth it for a project that is no longer maintained, according to the README. So I can do nothing more than repeat myself by saying: 'It works as expected, after making this change change'. So I'll keep using my fork :)

Reply:
Yup understood, can you close this PR then ?